cmd/lambda/leases/main.go

```diff
@@ -64,6 +66,7 @@ var (
 	baseRequest              url.URL
 	//cognitoUserPoolId        string
 	//cognitoAdminName         string
+	user *api.User
```

Comment:
Can we put the validate functionality on the struct itself?  Then we can just call a function on that struct?